Kitodo/src/test/java/org/kitodo/production/interfaces/activemq/KitodoScriptProcessorIT.java

```diff
@@ -0,0 +1,440 @@
+/*
+ * (c) Kitodo. Key to digital objects e. V. <contact@kitodo.org>
+ *
+ * This file is part of the Kitodo project.
+ *
+ * It is licensed under GNU General Public License version 3 or later.
+ *
+ * For the full copyright and license information, please read the
+ * GPL3-License.txt file that was distributed with this source code.
+ */
+package org.kitodo.production.interfaces.activemq;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+
+import java.lang.reflect.Field;
+import java.util.ArrayList;
+import java.util.Collection;
+import java.util.Collections;
+import java.util.Enumeration;
+import java.util.List;
+
+import javax.jms.Destination;
+import javax.jms.JMSException;
+import javax.jms.MapMessage;
+
+import org.junit.jupiter.api.AfterAll;
+import org.junit.jupiter.api.BeforeAll;
+import org.junit.jupiter.api.Test;
+import org.kitodo.MockDatabase;
+import org.kitodo.SecurityTestUtils;
+import org.kitodo.data.database.beans.Process;
+import org.kitodo.production.services.ServiceManager;
+import org.kitodo.production.services.command.KitodoScriptService;
+
+public class KitodoScriptProcessorIT {
+    @BeforeAll
+    public void prepare() throws Exception {
+        MockDatabase.startNode();
+        MockDatabase.insertProcessesForWorkflowFull();
+        SecurityTestUtils.addUserDataToSecurityContext(ServiceManager.getUserService().getById(1), 1);
+    }
+
+    @AfterAll
```

Comment:
This should be executed after each test or?
```suggestion
    @AfterEach
```

Reply:
It doesn't matter here, as there is only one test included. As far as I can see, the database is only removed once for all other test classes, not for each test. That's why I would do the same here.